### PR TITLE
Container.py fix to allow get_details

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5460858c0457ca013d0ddf4f3ca4e0e428dbcd71:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 33c822448e2924555883b7683eb0fad61dcdd692:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/containers/container.py [hunk 1]
```diff
@@ -44,8 +45,18 @@ def navigate(self, detail=True):
                     'containers_container_detail', context={
                         'container': self, 'pod': self.pod})
         else:
-            sel.force_navigate('containers_container', context={'container': self})
+            sel.force_navigate('containers_container',
+                               context={'container': self, 'pod': self.pod})
```
Comment (mfalesni) [inline cfme/containers/container.py]: Pod is in container, therefore it is useless to pass both.

**Comment by mfalesni** [comment]
Squash it to one commit, please.

**Comment by pavelzag** [comment]
There's one unwanted file there the provider.py How can I easily remove it from the PR?

**Comment by mfalesni** [comment]
If you want to remove the changes, tehn you have to check out that file from master and during squasihing, the changes will be removed:

you will do something like: git checkout <hash> path/to/file.py

it might work passing upstream/master as hash. Once you commit this and once you squash all the commits, the changes on that file will be removed.

**Comment by psav** [comment]
What test can we use to run this through PRT?

**Comment by pavelzag** [comment]
It was added as preparation for test_containers_data_integrity_properties

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by pavelzag** [comment]
Closing this PR. This PR covers the same thing without all the mess
https://github.com/ManageIQ/integration_tests/pull/3279

**Pull request closed without merging**